[Bug] Build Failure with ONNX Runtime Backend due to Logger Dependency

### Checklist

- [x] I have searched related issues but cannot get the expected help.
- [x] 2. I have read the [FAQ documentation](https://github.com/vbti-development/onedl-mmdeploy/tree/main/docs/en/faq.md) but cannot get the expected help.
- [x] 3. The bug has not been fixed in the latest version.

### Describe the bug

I encountered a build failure when attempting to compile only the model converter with the ONNX Runtime backend. The issue stems from a dependency on the `mmdeploy` logger within the ONNX Runtime operator implementation.

Specifically, the following line in `csrc/mmdeploy/backend_ops/onnxruntime/modulated_deform_conv/modulated_deform_conv.cpp`:

```cpp
// In file: csrc/mmdeploy/backend_ops/onnxruntime/modulated_deform_conv/modulated_deform_conv.cpp
// Line: 8
#include "mmdeploy/core/logger.h"
```

introduces a direct dependency on `spdlog` (through the `logger.h`).  When building the model converter separately (e.g., using `cmake -DCMAKE_CXX_COMPILER=g++-7 -DMMDEPLOY_TARGET_BACKENDS=ort -DONNXRUNTIME_DIR=${ONNXRUNTIME_DIR} ..`), the `spdlog` library is not correctly linked, leading to a compilation error. Since the standalone compilation is only targeted for model conversion, the logger is unnecessary.

**Steps to reproduce:**

1.  Configure CMake with the following command:
    ```bash
    cmake -DCMAKE_CXX_COMPILER=g++-7 -DMMDEPLOY_TARGET_BACKENDS=ort -DONNXRUNTIME_DIR=${ONNXRUNTIME_DIR} ..
    ```
2.  Attempt to build the project using `make`.

**Observed behavior:**

The build fails with errors indicating that `spdlog` is not found.
```
In file included from /code/onedl-mmdeploy/csrc/mmdeploy/backend_ops/onnxruntime/modulated_deform_conv/modulated_deform_conv.cpp:8:
/code/onedl-mmdeploy/csrc/mmdeploy/core/logger.h:6:10: fatal error: spdlog/spdlog.h: No such file or directory
    6 | #include <spdlog/spdlog.h>
      |          ^~~~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [csrc/mmdeploy/backend_ops/onnxruntime/CMakeFiles/mmdeploy_onnxruntime_ops_obj.dir/build.make:104: csrc/mmdeploy/backend_ops/onnxruntime/CMakeFiles/mmdeploy_onnxruntime_ops_obj.dir/modulated_deform_conv/modulated_deform_conv.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [CMakeFiles/Makefile2:130: csrc/mmdeploy/backend_ops/onnxruntime/CMakeFiles/mmdeploy_onnxruntime_ops_obj.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
```

**Expected behavior:**

The build should succeed when targeting only the ONNX Runtime backend for model conversion, without requiring the full `mmdeploy` dependency (specifically, `spdlog`).

**Workaround:**

Removing the problematic line (`#include "mmdeploy/core/logger.h"`) in `modulated_deform_conv.cpp` and (`target_link_libraries(${PROJECT_NAME} PRIVATE fmt)`) in `csrc/mmdeploy/backend_ops/onnxruntime/CMakeLists.txt#L29` allows the compilation to succeed.


### Reproduction

cmake -DCMAKE_CXX_COMPILER=g++-7 -DMMDEPLOY_TARGET_BACKENDS=ort -DONNXRUNTIME_DIR=${ONNXRUNTIME_DIR} ..

### Environment

```Shell
10/27 10:48:00 - mmengine - INFO - 

10/27 10:48:00 - mmengine - INFO - **********Environmental information**********
10/27 10:48:02 - mmengine - INFO - sys.platform: linux
10/27 10:48:02 - mmengine - INFO - Python: 3.10.12 (main, Nov  6 2024, 20:22:13) [GCC 11.4.0]
10/27 10:48:02 - mmengine - INFO - CUDA available: True
10/27 10:48:02 - mmengine - INFO - MUSA available: False
10/27 10:48:02 - mmengine - INFO - numpy_random_seed: 2147483648
10/27 10:48:02 - mmengine - INFO - GPU 0,1: Tesla V100-PCIE-32GB
10/27 10:48:02 - mmengine - INFO - CUDA_HOME: /usr/local/cuda
10/27 10:48:02 - mmengine - INFO - NVCC: Cuda compilation tools, release 11.8, V11.8.89
10/27 10:48:02 - mmengine - INFO - GCC: x86_64-linux-gnu-gcc (Ubuntu 11.4.0-1ubuntu1~22.04) 11.4.0
10/27 10:48:02 - mmengine - INFO - PyTorch: 2.1.2+cu118
10/27 10:48:02 - mmengine - INFO - PyTorch compiling details: PyTorch built with:
  - GCC 9.3
  - C++ Version: 201703
  - Intel(R) oneAPI Math Kernel Library Version 2022.2-Product Build 20220804 for Intel(R) 64 architecture applications
  - Intel(R) MKL-DNN v3.1.1 (Git Hash 64f6bcbcbab628e96f33a62c3e975f8535a7bde4)
  - OpenMP 201511 (a.k.a. OpenMP 4.5)
  - LAPACK is enabled (usually provided by MKL)
  - NNPACK is enabled
  - CPU capability usage: AVX512
  - CUDA Runtime 11.8
  - NVCC architecture flags: -gencode;arch=compute_50,code=sm_50;-gencode;arch=compute_60,code=sm_60;-gencode;arch=compute_70,code=sm_70;-gencode;arch=compute_75,code=sm_75;-gencode;arch=compute_80,code=sm_80;-gencode;arch=compute_86,code=sm_86;-gencode;arch=compute_37,code=sm_37;-gencode;arch=compute_90,code=sm_90
  - CuDNN 8.7
  - Magma 2.6.1
  - Build settings: BLAS_INFO=mkl, BUILD_TYPE=Release, CUDA_VERSION=11.8, CUDNN_VERSION=8.7.0, CXX_COMPILER=/opt/rh/devtoolset-9/root/usr/bin/c++, CXX_FLAGS= -D_GLIBCXX_USE_CXX11_ABI=0 -fabi-version=11 -fvisibility-inlines-hidden -DUSE_PTHREADPOOL -DNDEBUG -DUSE_KINETO -DLIBKINETO_NOROCTRACER -DUSE_FBGEMM -DUSE_QNNPACK -DUSE_PYTORCH_QNNPACK -DUSE_XNNPACK -DSYMBOLICATE_MOBILE_DEBUG_HANDLE -O2 -fPIC -Wall -Wextra -Werror=return-type -Werror=non-virtual-dtor -Werror=bool-operation -Wnarrowing -Wno-missing-field-initializers -Wno-type-limits -Wno-array-bounds -Wno-unknown-pragmas -Wno-unused-parameter -Wno-unused-function -Wno-unused-result -Wno-strict-overflow -Wno-strict-aliasing -Wno-stringop-overflow -Wno-psabi -Wno-error=pedantic -Wno-error=old-style-cast -Wno-invalid-partial-specialization -Wno-unused-private-field -Wno-aligned-allocation-unavailable -Wno-missing-braces -fdiagnostics-color=always -faligned-new -Wno-unused-but-set-variable -Wno-maybe-uninitialized -fno-math-errno -fno-trapping-math -Werror=format -Werror=cast-function-type -Wno-stringop-overflow, LAPACK_INFO=mkl, PERF_WITH_AVX=1, PERF_WITH_AVX2=1, PERF_WITH_AVX512=1, TORCH_DISABLE_GPU_ASSERTS=ON, TORCH_VERSION=2.1.2, USE_CUDA=ON, USE_CUDNN=ON, USE_EXCEPTION_PTR=1, USE_GFLAGS=OFF, USE_GLOG=OFF, USE_MKL=ON, USE_MKLDNN=ON, USE_MPI=OFF, USE_NCCL=1, USE_NNPACK=ON, USE_OPENMP=ON, USE_ROCM=OFF, 

10/27 10:48:02 - mmengine - INFO - TorchVision: 0.16.2+cu118
10/27 10:48:02 - mmengine - INFO - OpenCV: 4.10.0
10/27 10:48:02 - mmengine - INFO - MMEngine: 0.10.5
10/27 10:48:02 - mmengine - INFO - MMCV: 2.1.0
10/27 10:48:02 - mmengine - INFO - MMCV Compiler: GCC 9.3
10/27 10:48:02 - mmengine - INFO - MMCV CUDA Compiler: 11.8
10/27 10:48:02 - mmengine - INFO - MMDeploy: 1.3.1+80ad1ec
10/27 10:48:02 - mmengine - INFO - 

10/27 10:48:02 - mmengine - INFO - **********Backend information**********
10/27 10:48:02 - mmengine - INFO - tensorrt:    8.6.1
10/27 10:48:02 - mmengine - INFO - tensorrt custom ops: NotAvailable
10/27 10:48:02 - mmengine - INFO - ONNXRuntime: None
10/27 10:48:02 - mmengine - INFO - ONNXRuntime-gpu:     1.15.1
10/27 10:48:02 - mmengine - INFO - ONNXRuntime custom ops:      Available
10/27 10:48:02 - mmengine - INFO - pplnn:       None
10/27 10:48:02 - mmengine - INFO - ncnn:        None
10/27 10:48:02 - mmengine - INFO - snpe:        None
10/27 10:48:02 - mmengine - INFO - openvino:    None
10/27 10:48:02 - mmengine - INFO - torchscript: 2.1.2+cu118
10/27 10:48:02 - mmengine - INFO - torchscript custom ops:      NotAvailable
10/27 10:48:02 - mmengine - INFO - rknn-toolkit:        None
10/27 10:48:02 - mmengine - INFO - rknn-toolkit2:       None
10/27 10:48:03 - mmengine - INFO - ascend:      None
10/27 10:48:03 - mmengine - INFO - coreml:      None
10/27 10:48:03 - mmengine - INFO - tvm: None
10/27 10:48:03 - mmengine - INFO - vacc:        None
10/27 10:48:03 - mmengine - INFO - 

10/27 10:48:03 - mmengine - INFO - **********Codebase information**********
10/27 10:48:03 - mmengine - INFO - mmdet:       3.3.0
10/27 10:48:03 - mmengine - INFO - mmseg:       1.2.2
10/27 10:48:03 - mmengine - INFO - mmpretrain:  1.2.0
10/27 10:48:03 - mmengine - INFO - mmocr:       None
10/27 10:48:03 - mmengine - INFO - mmagic:      None
10/27 10:48:03 - mmengine - INFO - mmdet3d:     None
10/27 10:48:03 - mmengine - INFO - mmpose:      None
10/27 10:48:03 - mmengine - INFO - mmrotate:    None
10/27 10:48:03 - mmengine - INFO - mmaction:    None
10/27 10:48:03 - mmengine - INFO - mmrazor:     1.0.0
10/27 10:48:03 - mmengine - INFO - mmyolo:      None
```

### Error traceback

```Shell
In file included from /code/onedl-mmdeploy/csrc/mmdeploy/backend_ops/onnxruntime/modulated_deform_conv/modulated_deform_conv.cpp:8:
/code/onedl-mmdeploy/csrc/mmdeploy/core/logger.h:6:10: fatal error: spdlog/spdlog.h: No such file or directory
    6 | #include <spdlog/spdlog.h>
      |          ^~~~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [csrc/mmdeploy/backend_ops/onnxruntime/CMakeFiles/mmdeploy_onnxruntime_ops_obj.dir/build.make:104: csrc/mmdeploy/backend_ops/onnxruntime/CMakeFiles/mmdeploy_onnxruntime_ops_obj.dir/modulated_deform_conv/modulated_deform_conv.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [CMakeFiles/Makefile2:130: csrc/mmdeploy/backend_ops/onnxruntime/CMakeFiles/mmdeploy_onnxruntime_ops_obj.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
```

--- a/README.md
+++ b/README.md
@@ -354,11 +354,11 @@ This project is released under the [Apache 2.0 license](LICENSE).
 
 ## Projects in VBTI-development
 
-- [MMEngine](https://github.com/vbti-development/onedl-mmengine): Foundational library for training deep learning models.
-- [MMCV](https://github.com/vbti-development/onedl-mmcv): Foundational library for computer vision.
-- [MMPreTrain](https://github.com/vbti-development/onedl-mmpretrain): Pre-training toolbox and benchmark.
-- [MMDetection](https://github.com/vbti-development/onedl-mmdetection): Detection toolbox and benchmark.
-- [MMRotate](https://github.com/vbti-development/onedl-mmrotate): Rotated object detection toolbox and benchmark.
-- [MMSegmentation](https://github.com/vbti-development/onedl-mmsegmentation): Semantic segmentation toolbox and benchmark.
-- [MMDeploy](https://github.com/vbti-development/onedl-mmdeploy): Model deployment framework.
-- [MIM](https://github.com/vbti-development/onedl-mim): MIM installs VBTI packages.
+- [OneDL-MMEngine](https://github.com/vbti-development/onedl-mmengine): Foundational library for training deep learning models.
+- [OneDL-MMCV](https://github.com/vbti-development/onedl-mmcv): Foundational library for computer vision.
+- [OneDL-MMPreTrain](https://github.com/vbti-development/onedl-mmpretrain): Pre-training toolbox and benchmark.
+- [OneDL-MMDetection](https://github.com/vbti-development/onedl-mmdetection): Detection toolbox and benchmark.
+- [OneDL-MMRotate](https://github.com/vbti-development/onedl-mmrotate): Rotated object detection toolbox and benchmark.
+- [OneDL-MMSegmentation](https://github.com/vbti-development/onedl-mmsegmentation): Semantic segmentation toolbox and benchmark.
+- [OneDL-MMDeploy](https://github.com/vbti-development/onedl-mmdeploy): Model deployment framework.
+- [OneDL-MIM](https://github.com/vbti-development/onedl-mim): MIM installs VBTI packages.

--- a/csrc/mmdeploy/backend_ops/onnxruntime/CMakeLists.txt
+++ b/csrc/mmdeploy/backend_ops/onnxruntime/CMakeLists.txt
@@ -26,7 +26,5 @@ target_link_libraries(${PROJECT_NAME} PUBLIC ${PROJECT_NAME}_obj)
 mmdeploy_add_rpath(${PROJECT_NAME})
 add_library(mmdeploy::onnxruntime::ops ALIAS ${PROJECT_NAME})
 
-target_link_libraries(${PROJECT_NAME} PRIVATE fmt)
-
 set(_ORT_OPS_DIR ${CMAKE_SOURCE_DIR}/mmdeploy/lib)
 install(TARGETS ${PROJECT_NAME} DESTINATION ${_ORT_OPS_DIR})

--- a/csrc/mmdeploy/backend_ops/onnxruntime/modulated_deform_conv/modulated_deform_conv.cpp
+++ b/csrc/mmdeploy/backend_ops/onnxruntime/modulated_deform_conv/modulated_deform_conv.cpp
@@ -5,7 +5,6 @@
 #include <thread>
 #include <vector>
 
-#include "mmdeploy/core/logger.h"
 #include "modulated_deform_conv/modulated_deform_conv_cpu.h"
 #include "ort_utils.h"
 

